backend/maelstro/core/clone.py

```diff
@@ -47,6 +47,87 @@ def gn_dst(self) -> GnApi:
     def gs_dst(self) -> RestService:
         return self.geo_hnd.get_gs_service(self.dst_name, is_source=False)
 
+    def involved_resources(
+        self,
+        copy_meta: bool,
+        copy_layers: bool,
+        copy_styles: bool,
+    ) -> dict[str, list[dict[str, Any]]]:
+        self.copy_meta = copy_meta
+        self.copy_layers = copy_layers
+        self.copy_styles = copy_styles
+
+        with get_georchestra_handler() as geo_hnd:
+            self.geo_hnd = geo_hnd
+
+            zipdata = self.gn_src.get_record_zip(self.uuid).read()
+            self.meta = Meta(zipdata)
+
+            resources: dict[str, list[dict[str, Any]]] = {
+                "metadata": [],
+                "data": [],
+            }
+
+            src_gn_info = self.geo_hnd.get_service_info(
+                self.src_name, is_source=True, is_geonetwork=True
+            )
+            src_gn_url = src_gn_info["url"]
+            dst_gn_info = self.geo_hnd.get_service_info(
+                self.dst_name, is_source=False, is_geonetwork=True
+            )
+            dst_gn_url = dst_gn_info["url"]
+
+            resources["metadata"].append(
+                {
+                    "src": src_gn_url,
+                    "dst": dst_gn_url,
+                    "metadata": (
+                        [
```

Comment:
We don't need a list here, there is only one dataset

Reply:
This is the case today, but we may have multiple ones at some time.

Reply:
Then I don't get it. There is an outer list with one (and later on potentially multiple) metadatas, but here is the "inner list" there can only be one metadata per metadata. Maybe there is a misunderstanding between us concerning nested loops.

Reply:
I made the same as for geoserver, one list by source server and a inner list for metadata.